lib/src/common.dart

```diff
@@ -30,7 +30,7 @@ String generateHash(Object data) {
   return hex.encode(Hash.sha256(bytes));
 }
 
-Uint8List _serializeProto(GeneratedMessage data) {
+Uint8List serializeProto(GeneratedMessage data) {
```

Comment:
Not related to this PR, but why do we need these lines?

```dart
if (data.runtimeType == proto.BirthDate) {
  final value = data as proto.BirthDate;
  data = proto.BirthDate(value: Timestamp()..seconds = value.value.seconds);
}
```

Reply:
The hashes didn't match on dart and clients when comparing timestamps. They seem to write it differently